pkg/detectors/zendeskapi/zendeskapi.go

```diff
@@ -21,8 +21,8 @@ var _ detectors.Detector = (*Scanner)(nil)
 var (
 	client = common.SaneHttpClient()
 
-	token  = regexp.MustCompile(detectors.PrefixRegex([]string{"zendesk"}) + `([A-Za-z0-9_-]{40})`)
-	email  = regexp.MustCompile(`\b([a-zA-Z-0-9-]{5,16}\@[a-zA-Z-0-9]{4,16}\.[a-zA-Z-0-9]{3,6})\b`)
+	token  = regexp.MustCompile(`([A-Za-z0-9_-]{40})`)
+	email  = regexp.MustCompile(`\b(\b[A-Za-z0-9._%+-]+@[A-Za-z0-9.-]+\.[A-Z|a-z]{2,7}\b)\b`)
```

Comment:
@ankushgoel27 any reason why we need back to back `\b`s here?

Reply:
Also, why `[A-Z|a-z]`? Is that meant to be uppercase OR lowercase `(A-Z|a-z)`? Regardless, I think `[A-Za-z]` would be fine.

Reply:
Fixed email regex. I believe it's ready for merge.
Thanks for pointing it out.

Reply:
@rgmz can we merge this please

Reply:
I'm just a random guy.

Reply:
@zricethezav can we merge this please

Reply:
@zricethezav can we merge this please